grudge/dof_desc.py

```diff
@@ -154,11 +170,12 @@ def __init__(self, domain_tag, quadrature_tag=None):
             :class:`~meshmode.mesh.BTAG_PARTITION`,
             or *None* to indicate that the geometry is not yet known.
 
-        :arg quadrature_tag:
-            *None* to indicate that the quadrature grid is not known, or
-            :class:`QTAG_NONE` to indicate the use of the basic discretization
-            grid, or a string to indicate the use of the thus-tagged quadratue
-            grid.
+        :arg discretization_tag:
+            *None* to indicate that the discretization grid is not known,
```

Comment:
> not known

What would that mean? (FWIW, I think `None`/"not specified" should mean "base" as the logic below says.)

Reply:
Done: https://github.com/inducer/grudge/pull/83/commits/a70b9d6f0b904382bed836331f6bb043cb6ea297